edb/server/main.py

```diff
@@ -469,6 +479,16 @@ def _protocol_version(
     click.option(
         '--default-database-user', type=str, default=getpass.getuser(),
         help='the name of the default database owner'),
+    click.option(
+        '--init-command', metavar="QUERIES",
```

Comment:
I still like `--bootstrap-command` better, because it signifies that this would _only_ be executed during bootstrap.  To clarify things, we can rename `--bootstrap` to `--bootstrap-only`.

Reply:
Sounds good.